lib/src/command/outdated.dart

```diff
@@ -263,9 +358,13 @@ Future<void> _outputJson(List<_PackageDetails> rows) async {
       .convert({'packages': rows.map((row) => row.toJson()).toList()}));
 }
 
-Future<void> _outputHuman(List<_PackageDetails> rows,
-    Future<List<_FormattedString>> Function(_PackageDetails) marker,
-    {@required bool useColors, @required bool includeDevDependencies}) async {
+Future<void> _outputHuman(
+  List<_PackageDetails> rows,
+  Future<List<_FormattedString>> Function(_PackageDetails) marker, {
+  @required bool useColors,
+  @required bool includeDevDependencies,
+  @required bool upgradableFailed,
```

Comment:
This seems too specific.

Perhaps we could take a list of warnings to print?

(Current solution is fine for now)